rdt/transformers/__init__.py

```diff
@@ -35,18 +46,44 @@ def _import_addons():
             for transformer in transformers:
                 module = transformer.rsplit('.', 1)[0]
                 if module not in sys.modules:
-                    importlib.import_module(module)
+                    module = importlib.import_module(module)
+                    for name, obj in inspect.getmembers(module):
+                        # Inject the addon transformer into this scope
+                        if inspect.isclass(obj) and issubclass(
+                                obj, BaseTransformer) and obj is not BaseTransformer:
+                            globals()[name] = obj
+                            __all__.append(name)
```

Comment:
What happens here if there are addons with the same name? Is it possible for us to import them by module and name?

Reply:
With the current code, if there are two add-ons with the same name, only the last add-on would be available to be imported from `rdt.transformers`. However, you could still import it by the full name if you wanted to (with `rdt.transformers.addons.X`).

Reply:
Should we just import them by full name here? Idk if it makes sense to overwrite other transformers

Reply:
You mean to just add the add-ons manually at the top of the file? I thought we didn't want to explicitly add them as imports since they're add-ons and not always imported.